R/pecs.R

```diff
@@ -78,12 +78,11 @@ pecs.list = function(x, measure = c("graf", "logloss"), times, n, eps = NULL, ta
   if (is.null(eps)) {
     eps = if (measure == "graf") 1e-3 else 1e-15
   } else {
-    assertNumeric(eps, lower = -1, upper = 1)
+    assert_numeric(eps, lower = -1, upper = 1)
   }
 
-  assert(all(sapply(x, function(y) !is.null(y$model))),
-    "x must be a list of trained survival learners")
-  assertClass(task, "TaskSurv")
+  assert_learners(x)
```

Comment:
the `assert_learners()` does not check if they are trained as well! and we need this here

Reply:
I've added the check back